Content.Shared/Item/SharedItemSystem.cs

```diff
@@ -83,14 +86,7 @@ private void OnHandInteract(EntityUid uid, ItemComponent component, InteractHand
 
     protected virtual void OnStackCountChanged(EntityUid uid, ItemComponent component, StackCountChangedEvent args)
     {
-        if (!TryComp<StackComponent>(uid, out var stack))
-            return;
 
-        if (!_prototype.TryIndex<StackPrototype>(stack.StackTypeId, out var stackProto) ||
-            stackProto.ItemSize is not { } size)
-            return;
-
-        SetSize(uid, args.NewCount * size, component);
     }
```

Comment:
Is this a dummy method now?

Reply:
there's an inherited method on the client

Reply:
whacky